psydac/linalg/tests/test_kron_direct_solver.py

```diff
@@ -76,20 +73,29 @@ def matrix_to_sparse(A):
 def random_matrix(seed, space):
     A = StencilMatrix(space, space)
     p = space.pads[0]
+    dtype=space.dtype
```

Comment:
Please add empty spaces before and after the equal sign.

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)